python/hawkey/tests/tests/test_reldep.py

```diff
@@ -35,20 +35,15 @@ def test_basic(self):
         reldep = requires[0]
         self.assertEqual(str(reldep), "P-lib >= 3")
 
-    def test_unicode(self):
```

Comment:
We should use some RPM parser for checking name correctness, but it's not exposed and that's not what libsolv does when parsing repomd